src/IntelligentRoutingScreens/IntelligentRoutingAnalytics.res

```diff
@@ -393,7 +393,7 @@ let make = () => {
 
   <PageLoaderWrapper screenState={screenState}>
     <div
-      className="absolute z-50 top-76-px left-0 w-full py-3 px-10 bg-orange-50 flex justify-between items-center">
+      className="absolute z-10 top-76-px left-0 w-full py-3 px-10 bg-orange-50 flex justify-between items-center">
```

Comment:
this change will cause overlap issue with the top right dropdown in the dual y-axis graph

Reply:
make it z-20 to resolve this